app/helpers/application_helper/toolbar/x_vm_cloud_center.rb

```diff
@@ -129,14 +129,21 @@ class ApplicationHelper::Toolbar::XVmCloudCenter < ApplicationHelper::Toolbar::B
   ])
   include ApplicationHelper::Toolbar::Cloud::InstanceOperationsButtonGroupMixin
   button_group('vm_access', [
-    button(
-      :cockpit_console,
-      'pficon pficon-screen fa-lg',
-      N_('Open a new browser window with Cockpit for this VM.  This requires that Cockpit is pre-configured on the VM.'),
-      N_('Web Console'),
-      # :image   => "cockpit",
-      :url   => "launch_cockpit",
-      :klass => ApplicationHelper::Button::CockpitConsole
-    )
+    select(
+      :vm_remote_access_choice,
+      'fa pficon-screen fa-lg',
+      N_('VM Remote Access'),
+      N_('Access'),
+      :items => [
+        button(
+          :vm_webmks_console,
```

Comment:
Are you dropping the `cockpit_console` intentionally here?

Reply:
Yes. It was never implemented, only the button was here (always disabled). I guess it was a copy-paste when adding vCloud provider. Best to remove it now and add later should it be needed.

Reply:
Okay for me, but I'd do a separate commit :wink:

Reply:
Done, I've left the cock
pit in code and only added TODO to remove it.